bay/plugins/build.py

```diff
@@ -60,18 +60,31 @@ def pre_start(self, host, instance, task):
                     host.client.inspect_volume(name)
                 except NotFound:
                     # Aha! Build it!
-                    Builder(
-                        host,
-                        providers[name],
-                        self.app,
-                        parent_task=task,
-                        logfile_name=self.app.config.get_path(
-                            'bay',
-                            'build_log_path',
+                    try:
+                        logfile_name = self.app.config.get_path(
+                                'bay',
+                                'build_log_path',
+                                self.app,
+                            )
+                        Builder(
+                            host,
+                            providers[name],
                             self.app,
-                        ),
-                        verbose=True,
-                    ).build()
+                            parent_task=task,
+                            logfile_name=logfile_name,
+                            verbose=True,
+                        ).build()
+                    except BuildFailureError:
+                        click.echo(RED("Build failed! Last 15 lines of log:"))
+                        # TODO: More efficient tailing
```

Comment:
Since this is now in two places it might be nice to have it as a `util` function, but I'm not going to block on that.